scripts/setup-fix.bat

```diff
@@ -0,0 +1,11 @@
+if ! [ -x "$(command -v python)" ];
+then 
+	echo Python environment notfound..
+else 
+	echo Python environment found..
+fi
+python --version
+cp -R ../kubernetes/base/* ../kubernetes/
```

Comment:
what's the reason to do the 'cp -R' above?

Reply:
> what's the reason to do the 'cp -R' above?

"When we clone submodule with git the folders in the /kubernetes/base is supposed to be symbolically linked in the /kubernetes directory. In Windows this is not done. For a Windows developer, we’ll need to have a script copying all the base/* folders into */. "

^ is what @micw523 requested in his issue. so 'cp -R' does the job.